src/unit/test_crc64.c

```diff
@@ -0,0 +1,31 @@
+#include <stdio.h>
+#include "../crc64.h"
+
+#include "test_help.h"
+
+extern uint64_t _crc64(uint_fast64_t crc, const void *in_data, const uint64_t len);
+
+int test_crc64(int argc, char *argv[], int flags) {
```

Comment:
For symmetry purpose

```suggestion
int test_crc64(int argc, char **argv, int flags) {
```